### DISCUSSION TITLE
Unistalled your app for these reasons

### DISCUSSION THREAD

**Discussion by serrq** [trakt/showly#921] [General]
Please remove these unnecessary permissions.
For privacy and app behaviour reasons. I always want staying in control about my apps can do autonomously. Change network connectivity and Adv ID are not in my radar.

![1000134286](https://github.com/user-attachments/assets/b605057f-b01d-472b-962b-63da4f36bcd5)

**Comment by michaldrabik** [comment]
Thanks for pointing out. Advertisment ID is indeed a leftover. Will be removed next version - but it is used for nothing basically.

The CHANGE_NETWORK_STATE permission is required by Android to perform background networking - for example when syncing data. So this one will need to stay: https://developer.android.com/develop/connectivity/network-ops/reading-network-state#restrictions

**Reply by serrq** [reply]
Other apps perform sync in background without this permission.

There is also a permission in battery tab where a user can allow "background" activity for a certain app, selectively.

No needed.

**Reply by michaldrabik** [reply]
The Android documentation is pretty clear on that so I am reluctant to remove it TBH. Also it has nothing to do with your data privacy really.

As I am reading it is only needed to allow app to request networks connection in some cases:

> Apps with this permission let the system try to bring up a network that isn't up, such as the mobile network when the device is connected to a Wi-Fi network.

The permission itself has
> Protection level: normal

which means it possess no risk to user's data or privacy.

**Comment by serrq** [comment]
There is nothing to discuss with an uncollaborrative developer.

Unistalled (permanently)
